src/test/java/org/dataloader/DataLoaderMapBatchLoaderTest.java

```diff
@@ -1,184 +0,0 @@
-package org.dataloader;
-
-import org.junit.jupiter.api.Test;
-
-import java.util.ArrayList;
-import java.util.Collection;
-import java.util.HashMap;
-import java.util.List;
-import java.util.Map;
-import java.util.concurrent.CompletableFuture;
-import java.util.concurrent.ExecutionException;
-import java.util.concurrent.atomic.AtomicInteger;
-
-import static java.util.Arrays.asList;
-import static java.util.Collections.singletonList;
-import static org.awaitility.Awaitility.await;
-import static org.dataloader.DataLoaderFactory.newDataLoader;
-import static org.dataloader.DataLoaderOptions.newOptions;
-import static org.dataloader.fixtures.TestKit.futureError;
-import static org.dataloader.fixtures.TestKit.listFrom;
-import static org.dataloader.impl.CompletableFutureKit.cause;
-import static org.hamcrest.MatcherAssert.assertThat;
-import static org.hamcrest.Matchers.equalTo;
-import static org.hamcrest.Matchers.instanceOf;
-import static org.hamcrest.Matchers.is;
-
-/**
- * Much of the tests that related to {@link MappedBatchLoader} also related to
- * {@link org.dataloader.BatchLoader}.  This is white box testing somewhat because we could have repeated
- * ALL the tests in {@link org.dataloader.DataLoaderTest} here as well but chose not to because we KNOW that
- * DataLoader differs only a little in how it handles the 2 types of loader functions. We choose to grab some
- * common functionality for repeat testing and otherwise rely on the very complete other tests.
- */
-public class DataLoaderMapBatchLoaderTest {
-
-    MappedBatchLoader<String, String> evensOnlyMappedBatchLoader = (keys) -> {
-        Map<String, String> mapOfResults = new HashMap<>();
-
-        AtomicInteger index = new AtomicInteger();
-        keys.forEach(k -> {
-            int i = index.getAndIncrement();
-            if (i % 2 == 0) {
-                mapOfResults.put(k, k);
-            }
-        });
-        return CompletableFuture.completedFuture(mapOfResults);
-    };
-
-    private static <K, V> DataLoader<K, V> idMapLoader(DataLoaderOptions options, List<Collection<K>> loadCalls) {
-        MappedBatchLoader<K, V> kvBatchLoader = (keys) -> {
-            loadCalls.add(new ArrayList<>(keys));
-            Map<K, V> map = new HashMap<>();
-            //noinspection unchecked
-            keys.forEach(k -> map.put(k, (V) k));
-            return CompletableFuture.completedFuture(map);
-        };
-        return DataLoaderFactory.newMappedDataLoader(kvBatchLoader, options);
-    }
-
-    private static <K, V> DataLoader<K, V> idMapLoaderBlowsUps(
-            DataLoaderOptions options, List<Collection<K>> loadCalls) {
-        return newDataLoader((keys) -> {
```

Comment:
I'm not sure this was testing a `MappedBatchLoader` - we would have needed `newMappedDataLoader` for this.